docs/graphs/graph-queries/traversal-queries/index.md

```diff
@@ -3,102 +3,150 @@ sidebar_position: 1
+
+Graph traversal queries can help you find the shortest path between two nodes, discover patterns or clusters within the graph, or analyze the connectedness and centrality of nodes. By using graph traversal queries, you can efficiently process large and complex data sets with numerous connections and uncover insights that would be difficult to achieve with traditional relational databases.
+
+## Understanding Graph Traversal
+
+A traversal starts at a specific document or node, referred to as the `startVertex`, and follows all edges connected to it. It then visits all the connected nodes (also called vertices) and continues following their connected edges. This process is repeated iteratively for a predefined number of iterations, defined by the minimum (`min`) and maximum (`max`) depth.
+
+During the traversal process, every visited vertex between the min and max depth is returned as a set of three items:
+
+- The visited vertex: the current node being traversed.
+- The edge pointing to it: the connection between the current node and the previously visited node.
+- The complete path from the `startVertex` to the visited vertex: this is represented as an object with two attributes, `edges` and `vertices`. Both are lists of the corresponding elements in the path, and they are sorted in traversal order. In the vertices list, the first element is the `startVertex`, and the last element is the visited vertex. In the `edges` list, the nth element connects the nth element with the (n+1)th element in the `vertices` list.
 
 ## Syntax
 
-There are two slightly different syntaxes for traversals in C8QL, one for
+When writing a traversal query, you can either use a named graph or a set of edge collections (anonymous graph).
 
-- named graphs and another to
-- specify a [set of edge collections](#working-with-collection-sets)
+### Syntax for Named Graphs
 
-### Working with named graphs
+Use this syntax if you have created a named graph, which is entered as `graphName`.
 
-```js
+```sql
 [WITH collection1[, collection2[, ...collectionN]]]
 FOR vertex[, edge[, path]]
   IN [min[..max]]
   OUTBOUND|INBOUND|ANY startVertex
   GRAPH graphName
+  PRUNE pruneCondition
   [OPTIONS options]
+  [FILTER filterCondition]
+  [LET variableDefinitions]
+  [COLLECT aggregation]
+  [SORT sortOrder]
+  [LIMIT offset, count]
+  [RETURN returnExpression]
 ```
 
-`WITH`:
+### Syntax for Collection Sets
 
-- **collections** (collection, `repeatable`): list of collections that will be involved in the traversal
+Instead of `GRAPH graphName`, you can specify a list of edge collections (anonymous graph). The involved vertex collections are determined by the edges of the given edge collections. The rest of the behavior is similar to the named version.
```

Comment:
"The involved vertex collections" sounds funny.
Shouldn't it be "The vertex collections involved"

The rest of the behavior... -> Other than that, the behavior... ?